app-frontend/src/app/pages/projects/edit/advancedcolor/adjust/adjust.controller.js

```diff
@@ -252,6 +252,14 @@ export default class ProjectsColorAdjustController {
         }
     }
 
+    onGammaFilterChange(id, val, options={}) {
```

Comment:
`options={}` => `options = {}` according to our linting setup (also on line 271)